packages/web-components/fast-components/src/tooltip/README.md

```diff
@@ -1,4 +1,5 @@
 # fast-tooltip
+
```

Comment:
Prettier made this change. I guess it's fine

Reply:
yep! Thanks @m4thieulavoie